src/AniLink/PaintDrip.js

```diff
@@ -107,9 +113,9 @@ export default class PaintDrip extends Component {
 	}
 
 	slideIn = ({ length }, node, from) => {
-		new TimelineMax().from(node, length, {
+		gsap.timeline().from(node, length, {
```

Comment:
Looks like this can just be .from
```suggestion
		gsap.from(node, length, {
```

Reply:
Good catch! Updated